src/encoded/schemas/biosample.json

```diff
@@ -137,14 +137,14 @@
         }
     },
     "facets": {
-        "biosource_summary": {
+        "biosample_type": {
             "title": "Sample Type"
         },
         "biosource.cell_line_tier": {
             "title": "4DN Cell Line Tier"
         },
-        "biosource.biosource_type": {
-            "title": "Biosource Type"
+        "biosource.biosource_summary": {
```

Comment:
biosource_summary is actually a calc prop of biosample, so this should be just "biosource_summary" (like in columns below) instead of "biosource.biosource_summary". Sorry if my previous comment wasn't as clear as it could have been!